### PR TITLE
feat: useFullscreen hook支持默认参数。并且同步更新文档与demo。

### PR DESCRIPTION
**Pull request by XiaoDaiGua-Ray**
[[中文版模板 / Chinese template](https://github.com/InhiblabCore/vue-hooks-plus/tree/master/.github/PULL_REQUEST_TEMPLATE-zh-CN.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [x] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      useFullscreen hook Supports default parameters     |
| 🇨🇳 Chinese |    useFullscreen hook支持默认参数       |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Follow our [Code of Conduct](https://github.com/InhiblabCore/vue-hooks-plus/blob/master/CODE_OF_CONDUCT.md)
- [x] Read the [Contributing Guidelines](https://github.com/InhiblabCore/vue-hooks-plus/blob/master/CONTRIBUTING.md)
- [x] Read the [docs](https://inhiblabcore.github.io/docs/hooks)
- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: packages/hooks/src/useFullscreen/index.ts [hunk 1]
```diff
@@ -33,10 +33,11 @@ const useFullscreen = (
   const setState = (val: boolean) => {
     state.value = val
   }
+  const modelTraget = computed(() => getTargetElement(target) ?? document?.querySelector('html'))
```
Comment (NelsonYong) [inline packages/hooks/src/useFullscreen/index.ts:36]: const bindTarget =  getTargetElement(target, document?.querySelector('html'))
改一下名字，然后 getTargetElement 的第二个参数支持默认 default。 computed 可以不需要。

Comment (NelsonYong) [inline packages/hooks/src/useFullscreen/index.ts:36]: document?. 的 ？ 可以去掉

File: packages/hooks/src/useFullscreen/index.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ const useFullscreen = (
   /**
    * DOM element or ref
    */
-  target: BasicTarget,
+  target?: BasicTarget,
   options?: UseFullscreenOptions,
```
Comment (NelsonYong) [inline packages/hooks/src/useFullscreen/index.ts:25]: option 可以加多一个 defaultElment，兼容其他情况

File: packages/hooks/src/useFullscreen/index.ts [hunk 3]
```diff
@@ -33,10 +33,11 @@ const useFullscreen = (
   const setState = (val: boolean) => {
     state.value = val
   }
+  const modelTraget = computed(() => getTargetElement(target) ?? document?.querySelector('html'))
 
   const onChange = () => {
     if (screenfull.isEnabled) {
-      const el = getTargetElement(target)
+      const el = getTargetElement(modelTraget)
```
Comment (NelsonYong) [inline packages/hooks/src/useFullscreen/index.ts:40]: 为什么这里还要使用  getTargetElement 这个方法

File: packages/hooks/src/useFullscreen/index.ts (hunk 2 above)
Comment (XiaoDaiGua-Ray) [reply packages/hooks/src/useFullscreen/index.ts:25]: 觉得，如果需要配置，则会与默认html有点重复

File: packages/hooks/src/useFullscreen/index.ts (hunk 3 above)
Comment (XiaoDaiGua-Ray) [reply packages/hooks/src/useFullscreen/index.ts:40]: 每次调用 get 方法，避免dom ref卸载

File: packages/hooks/src/useFullscreen/index.ts (hunk 2 above)
Comment (NelsonYong) [reply packages/hooks/src/useFullscreen/index.ts:25]: 会存在一种场景，可能我不需要 html 为默认，我希望其他的

File: packages/hooks/src/useFullscreen/index.ts (hunk 3 above)
Comment (NelsonYong) [reply packages/hooks/src/useFullscreen/index.ts:40]: on change  的时机应该是避免这个问题，类似于 settimeout

**Review by NelsonYong — APPROVED** [review]

**Pull request merged**